README.md

```diff
@@ -1,32 +1,37 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+<!-- about  Microverse first project -->
 
-# Microver Capstone 1 
+ # Microverse First project  
+<img src="assert/desktop-version-screenshot.png">
 
-> Description the project.
+> Description of  the project.
 
+ This is a portfolio website 
 
 ## Built With
 
-- HTML,CSS
-- JAVASCRIPT
-- GIT
+- Html
+- Css
+- Github flow 
 
 ## Live Demo 
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://hedevedup.github.io/MicroversePortfolioProject/)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
+**Just Clone The Repo to get a Better Look At the Code**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
-
+**HTML**<b>
+**CSS**
+**Git And Github**
 ### Setup
+ 
 
 ### Install
```

Comment:
- [ ] Please write more descriptive steps of how to clone and set up your repository like you should write open terminal and write `git clone https://github.com/HeDevedUp/MicroCapstone1.git` and all the rest steps.